hibernate-core/src/main/antlr/sql-script.g

```diff
@@ -56,7 +56,7 @@ script
     ;
 
 statement
-	: { statementStarted(); } (statementPart)*  DELIMITER { statementEnded(); }
+	: { statementStarted(); } (statementPart)* DELIMITER (WS_CHAR)* { statementEnded(); }
```

Comment:
What is the reason for parsing trailing whitespace chars?

Reply:
It seems intuitive to me, though not 100% necessary for this PR. After reserving end-of-line chars, we have to remember to remove the end-of-line after last `;`, which seems too demanding to me.